operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md

```diff
@@ -83,6 +83,29 @@ export CSM_RELEASE=1.4.0
 export CSM_ARTI_DIR="/etc/cray/upgrade/csm/csm-${CSM_RELEASE}/tarball/csm-${CSM_RELEASE}"
 ```
 
+> **`NOTES:`**
+>
+> - If directory `/etc/cray/upgrade/csm/` is empty, create empty directory at same path. Download and extract CSM tarball to that directory.
+> - Update value of `CSM_ARTI_DIR` with newly created directory above.
+> - Ensure directory `/etc/cray/upgrade/csm/` is ceph mount using command as below:
+
+```bash
+mount | grep /etc/cray/upgrade/csm
+admin-tools@<hostname>=/ on /etc/cray/upgrade/csm type ceph (rw,relatime,name=admin-tools,secret=<hidden>,ms_mode=prefer-crc,acl,mon_addr=<ip:port>)
```

Comment:
I would remove the output line as it is a little confusing to include the output in the bash command block.

Reply:
Line removed. Thanks.